### PR TITLE
Test fill in values

### PR DESCRIPTION
**Pull request by Terica**
all tests pass

### CODE REVIEW

File: koans/about_asserts.rb [hunk 1]
```diff
@@ -13,28 +13,28 @@ def test_assert_truth
   # Enlightenment may be more easily achieved with appropriate
   # messages.
   def test_assert_with_message
-    assert false, "This should be true -- Please fix this"
+    assert true, "This should be true -- Please fix this"
```
Comment (fayep) [inline koans/about_asserts.rb:16]: This should not show up as a change once PR2 is merged

Comment (fayep) [reply koans/about_asserts.rb:16]: I think you may not have been working against an updated copy of the repo. A rebase would have made this cleaner

**Comment by Terica** [comment]
Thank you, Faye. You are showing me the mistake; twice. I can see how a
small blip leaves a big stain. Lot's of scrubbing; no time for that type of
work.
"I think you may not have been working against an updated copy of the repo.
A rebase would have made this cleaner" Got it.

To move faster I made a mess, but it passed. The versioning cannot be a
mess; of course. I think I will be cleaner from here out.

"This should not show up as a change once PR2 is merged" Understood

I'm getting this under my fingernails; very exciting.

**Comment by fayep** [comment]
You should set to squash and merge so that there is a single commit per PR.

**Comment by fayep** [comment]
I think you did that, just checking it was conscious

**Comment by Terica** [comment]
I did it because I want to give what you asked for.

**Pull request merged**